plugins/GSdx/GLLoader.cpp

```diff
@@ -247,7 +247,7 @@ namespace Emulate_DSA {
 
 	// Replace function pointer to emulate DSA behavior
 	void Init() {
-		fprintf(stderr, "DSA is not supported. Expect slower performance\n");
+		fprintf(stderr, "Direct State Access: is not supported! Expect slower performance\n");
```

Comment:
Not really , I just thought It would be better to match how the other messages look (texture barrier and viewpoint array.